drivers/adc/CMakeLists.txt

```diff
@@ -20,3 +20,4 @@ zephyr_library_sources_ifdef(CONFIG_ADC_NPCX		adc_npcx.c)
 zephyr_library_sources_ifdef(CONFIG_USERSPACE		adc_handlers.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_CC32XX		adc_cc32xx.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_EMUL		adc_emul.c)
+zephyr_library_include_directories(${CMAKE_CURRENT_LIST_DIR})
```

Comment:
Relative paths are relative to `CMAKE_CURRENT_SOURCE_DIR`, so you can also do.
```
zephyr_library_include_directories(.)
```

Reply:
done